memtable/vectorrep.cc

```diff
@@ -127,19 +147,40 @@ void VectorRep::MarkReadOnly() {
 }
 
 size_t VectorRep::ApproximateMemoryUsage() {
-  return sizeof(bucket_) + sizeof(*bucket_) +
-         bucket_->size() *
-             sizeof(
-                 std::remove_reference<decltype(*bucket_)>::type::value_type);
+  return bucket_size_.LoadRelaxed() *
+         sizeof(std::remove_reference<decltype(*bucket_)>::type::value_type);
+}
+
+void VectorRep::BatchPostProcess() {
```

Comment:
Hmm. Do you know why one of the WriteBatchInternal::InsertInto() overloads doesn't call inserter.PostProcess()? Seems suspicious